content/docs/command-reference/version.md

```diff
@@ -36,36 +34,36 @@ usage: dvc version [-h] [-q | -v]
 
 The detail of DVC version depends upon the way of installing DVC.
 
-- **Official release**: [These instructions](/doc/install) include ways to
-  install DVC using the official package stored in Python Packaging Authority.
-  We mark these official releases with tags on DVC's repository. Any issues
-  reported with the official build can be traced using the `_BASE_VERSION`
-  constant
-  [in our core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
-  For example `0.40.2`.
+- **Official release**: is a stable version of DVC (for example `1.7.4`), that
+  can be [installed](/doc/install) using the official package stored in
+  [Python Packaging Authority](https://www.pypa.io/) or with a binary. These
+  releases are marked with tags in the DVC repository (see
+  [changelog](https://github.com/iterative/dvc/releases) for release history).
+
+  If there are any issues reported with the official build, they can be traced
+  using the `_BASE_VERSION` constant in our
+  [core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
 
   > 💡 To only get this number, use `dvc -V`.
 
-- **Development version**: `pip install git+git://github.com/iterative/dvc` will
-  install DVC using the `master` branch of DVC's repository. Another way of
-  setting up the development version is to clone the repository and run
-  `pip install .`.
+- **Development version**: will
+  [install](/doc/user-guide/contributing/core#development-environment) DVC using
+  the master branch of DVC's repository. This allows us to to make changes and
+  run tests on DVC.
```

Comment:
Sorry. this is not a logical change. Just removing part of the sentence and leaving another one isn't good writing. Please read through this and rewrite it in a way that makes sense. What is this talking about? It's not about the dev version or what it does. It's about the possible component of the DVC version number printed by this command.

Please also review the next paragraph about the master branch (which was already tehre), it could be shorter and better written. Thanks